### PR TITLE
[SATURN-1834] Add fixed positioning to menu

### PR DESCRIPTION
**Pull request by s-rubenstein**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

**Review by Shakespeared — APPROVED** [review]
LGTM, tested in the UI

File: src/components/NewClusterModal.js [hunk 1]
```diff
@@ -773,6 +773,7 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
                   id,
                   isSearchable: false,
                   value: sparkMode,
+                  menuPosition: 'fixed',
```
Comment (breilly2) [inline src/components/NewClusterModal.js]: This introduces a different problem. If the pane is scrollable and you open the select, the options don't scroll with the pane. Is there another way we can fix this?

Comment (s-rubenstein) [reply src/components/NewClusterModal.js]: The only other way I could think of to do it was to put was `menuPlacement: 'top'`, but that's a change in behavior.

Comment (panentheos) [reply src/components/NewClusterModal.js]: We could add `flex: 'auto'` to the scrollable div to have it expand to fill the space, which should fix it.

Comment (s-rubenstein) [reply src/components/NewClusterModal.js]: I will try that!

Comment (s-rubenstein) [reply src/components/NewClusterModal.js]: That solves it, pushing up shortly.

**Pull request merged**